powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/UpdateRecoveryConfigRequest.java

```diff
@@ -28,10 +30,20 @@
 @Data
 public class UpdateRecoveryConfigRequest {
 
+    @Schema(description = "The identifier of the application")
+    @NotBlank(message = "Application ID must not be null when updating recovery code configuration")
     private String applicationId;
+
+    @Schema(description = "Whether recovery codes are enabled")
     private boolean activationRecoveryEnabled;
+
+    @Schema(description = "Whether recovery postcard is enabled")
     private boolean recoveryPostcardEnabled;
+
+    @Schema(description = "Whether multiple recovery codes per user are enabled")
```

Comment:
Nitpick to reflect the parameter name
```suggestion
    @Schema(description = "Whether multiple recovery codes per user are allowed")
```